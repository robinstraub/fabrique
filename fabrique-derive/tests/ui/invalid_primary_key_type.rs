use fabrique_derive::Factory;

#[derive(Factory)]
struct Anvil {
    #[fabrique(primary_key = "invalid")]
    id: u32,
    weight: u32,
}

fn main() {}