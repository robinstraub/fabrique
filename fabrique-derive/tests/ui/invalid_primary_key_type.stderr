error: Unknown value: `invalid` at primary_key
 --> tests/ui/invalid_primary_key_type.rs:4:1
  |
4 | struct Anvil {
  | ^^^^^^
